operators/admin/olm-restricted-networks.adoc

```diff
@@ -5,26 +5,24 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-For {product-title} clusters installed on restricted networks, also known as
-disconnected clusters, Operator Lifecycle Manager (OLM) by default cannot access
-the Red Hat-provided OperatorHub sources hosted on Quay.io because they require
-full Internet connectivity. However, cluster administrators can disable those
-remote sources, create local mirrors of that same content, and configure OLM to
-install and manage Operators from the local sources instead.
+For {product-title} clusters installed on restricted networks, also known as disconnected clusters, Operator Lifecycle Manager (OLM) by default cannot access the Red Hat-provided OperatorHub sources hosted remotely on Quay.io because those remote sources require full Internet connectivity. However, as a cluster administrator you can still enable your cluster to use OLM in a restricted network if you have a workstation with full Internet access, which is only used to prepare the mirrored content.
```

Comment:
A very small distinction about the end of the paragraph: is it the entire _workstation_ that is only used to prepare mirrored content, or is it only _Internet access_ that is only used to prepare mirrored content? 

If it's the former, then you might try, "...if you have a workstation that has full Internet access and is only used to prepare the mirrored content."

If it's the latter, then what you have is fine.

Reply:
Updated to split some sentences apart and re-LGTM'd offline.